newIDE/app/src/InstancesEditor/SelectedInstances.js

```diff
@@ -139,14 +174,44 @@ export default class SelectedInstances {
     objectButton: PIXI.Graphics,
     onMove: (event: PanMoveEvent) => void,
     onEnd: () => void,
+    onPanMove: (event: PanMoveEvent) => void,
+    onPanEnd: () => void,
```

Comment:
Could you group all the inputs into an object so that each input is named, the list here is now too long.
Something like:
```js
  _makeButton({objectButton, ...}: {| objectButton: PIXI.Graphics |}) { ... }
```